elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -309,22 +307,15 @@ export class PeerPool extends EventEmitter {
 	}
 
 	public async request(packet: P2PRequestPacket): Promise<P2PResponsePacket> {
-		const listOfPeerInfo = [...this._peerMap.values()].map(
-			(peer: Peer) => peer.peerInfo as P2PDiscoveredPeerInfo,
-		);
-		const selectedPeers = this._peerSelectForRequest({
```

Comment:
Should we change the selection function instead of completely bypassing selection function? because then the custom selection function for the request will not work. So let's modify the existing selection functions to simply return one incoming peerlist for both send and request and remove the height logic